system/Config/Services.php

```diff
@@ -96,7 +97,7 @@ public static function cache(\Config\Cache $config = null, $getShared = true)
 			return self::getSharedInstance('cache', $config);
 		}
 
-		if (! is_object($config))
+		if (!is_object($config))
```

Comment:
I think the ideal, in the long run, is a tool to correct formatting consistently, possibly as part of a PR.